OpenRA.Mods.Common/ServerTraits/MasterServerPinger.cs

```diff
@@ -68,10 +70,10 @@ public void Tick(S server)
 			{
 				var gs = new GameServer(server);
 				if (server.Settings.AdvertiseOnline)
-					UpdateMasterServer(server, gs.ToPOSTData(false));
+					UpdateMasterServer(server, gs.ToJson());
```

Comment:
Minor. I would pass the object. Let the Task call `ToJson`, only the task knows it will send 'application/json'.